tests/testthat/_snaps/bayes.md

```diff
@@ -451,15 +451,17 @@
       tune:::check_bayes_initial_size(5, 3, FALSE)
     Message
       ! There are 5 tuning parameters and 3 grid points were requested.
-      * There are more tuning parameters than there are initial points. This is likely to cause numerical issues in the first few search iterations.
+      * There are more tuning parameters than there are initial points. This is
+        likely to cause numerical issues in the first few search iterations.
 
 ---
 
     Code
       tune:::check_bayes_initial_size(5, 3, TRUE)
     Message
       ! There are 5 tuning parameters and 3 grid points were requested.
-      * There are more tuning parameters than there are initial points. This is likely to cause numerical issues in the first few search iterations.
+      * There are more tuning parameters than there are initial points. This is
```

Comment:
Can you explain to me why this line being split in 2?

Reply:
A bit buried in the docs, but from the [cli pkgdown site](https://cli.r-lib.org/reference/inline-markup.html?q=break#wrapping):

> Most cli containers wrap the text to width the container's width, while observing margins requested by the theme.

So, `rlang::inform` output is wrapped only if supported by the console, `cli::cli_bullet` output is wrapped in more contexts.🎁